main.go

```diff
@@ -23,6 +25,19 @@ func ShowInstances(instances []*Instance) {
 		"ICMP", "SSH", "HTTP", "HTTPS"})
 
 	for n, i := range instances {
+
+		if *filterRegex != "" {
+			regex, err := regexp.Compile(*filterRegex)
+			if err != nil {
+				fmt.Println(*filterRegex)
+				log.Fatal("Bad regex filter.")
```

Comment:
In general the whole PR looks good. Only comment is on this line. As a rule, in go, you really want to show the user the value of `err` whenever possible. Otherwise, they might not be able to tell what's wrong. The usual way I would achieve this is as so:

```golang
log.Fatalf("Regex does not compile: %v", err)
```

Or maybe

```
log.Fatalf("Failed to compile regex %q: %v", *filterRegex, err)
```

A good error message makes it clear what failed and why. `Bad regex filter` leaves it ambiguous what the program was trying to do with the regex when it failed.